src/wifi_service.c

```diff
@@ -625,10 +636,14 @@ static void connect_wifi_with_ssid(const char *ssid, jvalue_ref req_object, luna
 			settings->passkey = strdup(passkey_buf.m_str);
 			jstring_free_buffer(passkey_buf);
 		}
-		else if (jobject_get_exists(security_obj, J_CSTR_TO_BUF("enterpriseSecurity"), &enterprise_security_obj))
+		else if (jobject_get_exists(security_obj, J_CSTR_TO_BUF("enterpriseSecurity"), &enterprise_security_obj) &&
+			 jobject_get_exists(enterprise_security_obj, J_CSTR_TO_BUF("identityEAP"), &identity_obj) &&
+			 jobject_get_exists(enterprise_security_obj, J_CSTR_TO_BUF("passKey"), &passkey_obj))
 		{
-			LSMessageReplyCustomError(service_req->handle, service_req->message, "Not implemented");
-			goto cleanup;
+			identity_buf = jstring_get(identity_obj);
+			settings->identity = strdup(identity_buf.m_str);			
+			passkey_buf = jstring_get(passkey_obj);
+			settings->passkey = strdup(passkey_buf.m_str);
```

Comment:
Like this? https://github.com/Herrie82/webos-connman-adapter/commit/40c295d3150faa0fe45e958ab153330a549ce9e5